### PR TITLE
fix the mixed up git issue and upload the lastest version of game on master

### PR DESCRIPTION
**Pull request by tyeung2017**
relates #11

### CODE REVIEW

**Review by KatiaAsh — APPROVED** [review]
:video_game:

**Review by Wyrd00 — APPROVED** [review]

File: game.js [hunk 1]
```diff
@@ -44,7 +47,31 @@ function balloonKnot(x, y){
   ctx.fill();
 }
 
+function randomizer(num){
+  return Math.floor(Math.random() * num);
+}
+
+function colorPick(){ //yeah, we know having side effect to change global variable is not ideal
+  var pickColor = randomizer(3);
+  var result = pickColor===3?2:pickColor;
```
Comment (Karyum) [inline game.js:56]: es6 on frontend

File: game.js [hunk 2]
```diff
@@ -44,7 +47,31 @@ function balloonKnot(x, y){
...
+  switch (colorChart[result]) {
+    case 'gold':
+      breakPoint = randomizer(10) + 110;
+      // console.log(breakPoint);
```
Comment (Karyum) [inline game.js]: delete those console logs

**Review by Karyum — CHANGES_REQUESTED** [review]

File: game.js (hunk 1 above)
Comment (Wyrd00) [reply game.js:56]: I don't think ternary is es6.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator

**Review by KatiaAsh — APPROVED** [review]

File: game.js (hunk 1 above)
Comment (Karyum) [reply game.js:56]: my bad ! sorry

Comment (tyeung2017) [reply game.js:56]: watching you mate ! watching you !

**Pull request merged**